### ISSUE TITLE
Submit to stdlib?

### ISSUE THREAD

**Issue by paravoid** [drbild/sslpsk#18]
Thanks for your efforts in authoring and maintaining this module!

I was wondering whether this should be something that exists in Python's stdlib and if you've thought about that. I found [issue 19084](https://bugs.python.org/issue19084), which is very old and was resolved due to inactivity. Noone objected it on philosophical grounds though, and you seem to have done most of the work for it, so perhaps it'd be worth reviving that issue and continuing that conversation?

**Comment by drbild** [comment]
Yes, getting PSK support merged upstream would be far preferable to continuing to develop hacks for each new release of Python!

I'll try to make time to submit something in the next month or so. But if you want to submit something earlier (based on this code or your work in #19), please feel free!

**Comment by paravoid** [comment]
This is a couple of months old but I just noticed it: https://github.com/python/cpython/pull/103181
